docker/Dockerfile.geth

```diff
@@ -0,0 +1,47 @@
+FROM alpine:3.21.2 AS build
```

Comment:
Hope Alpine's musl doesn't introduce any linking problems :thinking:

Reply:
My thoughts exactly, but it seemed to work with libclang even rocksDB had no complains so that makes me happy.